foresight/model/test_run_result.py

```diff
@@ -0,0 +1,19 @@
+class TestRunResult:
```

Comment:
Abstract class 

```python
ABC = abc.ABCMeta('ABC', (object,), {})

class TestRunResult(ABC):
```

Reply:
This class is not abstract. It is initialized whilst sending finish test run event.

Reply:
fixed.

Reply:
Still not abstract class